### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#141] [Esercizi - Bertini] [answered]
![Es 3](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/a6b80bbd-9c5f-4818-a81e-4d460feea964)

**Comment by daromacarlo** [comment]
![Immagine 2023-10-14 181919](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/c93fd5f7-dfa5-4d68-b213-edb1e57469fb)

**Comment by CiottoloMaggico** [comment, marked as answer]
# Soluzione confermata in classe
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/bbc77ab3-2a33-4674-b812-3801594bc0fc)

**Comment by daromacarlo** [comment]
@CiottoloMaggico allora il mio è sbagliato, k lo intendeva fissato
